test/lib/usd/translators/testUsdExportDisplacement.py

```diff
@@ -0,0 +1,130 @@
+from maya import standalone
+
+import os
+import unittest
+
+import fixturesUtils
+
+
+class testUsdExportDisplacementShaders(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        inputPath = fixturesUtils.setUpClass(__file__)
+
+        mayaFile = os.path.join(inputPath, 'UsdExportDisplacementTest',
+            'SimpleDisplacement.ma')
+        cmds.file(mayaFile, force=True, open=True)
+
+        # Export to USD.
+        usdFilePath = os.path.abspath('SimpleDisplacement.usda')
+        cmds.mayaUSDExport(mergeTransformAndShape=True, file=usdFilePath,
+            shadingMode='useRegistry', convertMaterialsTo='UsdPreviewSurface',
+            materialsScopeName='Materials')
+
+        cls._stage = Usd.Stage.Open(usdFilePath)
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    def testStageOpens(self):
+        '''
+        Tests that the USD stage was opened successfully.
+        '''
+        self.assertTrue(self._stage)
+
+    def testExportDisplacementShaders(self):
+        '''
+        Tests that exporting multiple Maya planes with varying displacement
+        setups results in the expected USD data:
+        '''
+
+        expected = [
+            # Mesh, Blinn, Image, Channel
+            ("13", "3", "RGBA", "r"),
+            ("14", "4", "RGBA", "g"),
+            ("15", "5", "RGBA", "b"),
+            ("16", "6", "RGBA", "a"),
+            ("17", "19", "RGBA", "a"),
+            ("9", "7", "RGB", "r"),
+            ("10", "8", "RGB", "g"),
+            ("11", "9", "RGB", "b"),
+            ("12", "10", "RGB", "a"),
+            ("20", "20", "RGB", "a"),
+            ("5", "11", "MonoA", "r"),
+            ("6", "12", "MonoA", "g"),
+            ("7", "13", "MonoA", "b"),
+            ("8", "14", "MonoA", "a"),
+            ("18", "21", "MonoA", "a"),
+            ("1", "15", "Mono", "r"),
+            ("2", "16", "Mono", "g"),
+            ("3", "17", "Mono", "b"),
+            ("4", "18", "Mono", "a"),
+            ("19", "22", "Mono", "a")
+        ]
+
+        for mesh_suffix, blinn_suffix, image, channel in expected:
+
+            mesh_prim = self._stage.GetPrimAtPath('/pPlane%s' % mesh_suffix)
+            self.assertTrue(mesh_prim)
+
+            # Validate the Material prim bound to the Mesh prim.
+            self.assertTrue(mesh_prim.HasAPI(UsdShade.MaterialBindingAPI))
+            mat_binding = UsdShade.MaterialBindingAPI(mesh_prim)
+            material = mat_binding.ComputeBoundMaterial()[0]
+            self.assertTrue(material)
+            materialPath = material.GetPath().pathString
+            self.assertEqual(materialPath,
+                             '/pPlane%s/Materials/blinn%sSG' % (mesh_suffix,
+                                                                blinn_suffix))
+
+            # We expect two outputs on the material: surface + displacement
+            mat_outs = material.GetOutputs()
+            self.assertEqual(len(mat_outs), 3)
```

Comment:
We do indeed only expect two *connected* outputs, but the fact that we're testing that we get three outputs back might be confusing, and we're not really verifying the expectation. We'd get three outputs back on a `Material` with no connections authored as well.

It might make it a little clearer to do something like this:

```Python
self.assertTrue(material.ComputeSurfaceSource())
self.assertTrue(material.ComputeDisplacementSource())
self.assertFalse(material.ComputeVolumeSource())
```